api-server/src/events/dto/eventDTO.ts

```diff
@@ -97,6 +109,21 @@ export default class EventDTO {
   @ApiProperty()
   venue: VenueModel;
 
+  @ApiProperty({
+    type: () => PartnerDTO,
+    nullable: true,
+    required: false,
+    description: 'The partner that owns this event',
+    example: {
+      id: '123e4567-e89b-12d3-a456-426614174000',
+      name: 'TechCorp Inc.',
+      logo_url: 'https://example.com/logo.png',
+      createdAt: '2024-01-15T10:30:00.000Z',
+      updatedAt: '2024-01-15T10:30:00.000Z',
+    },
+  })
+  owning_partner?: PartnerDTO;
+
```

Comment:
What do we do with `reviewed_by_org` -- mark it as deprecated?

Reply:
yeah, i was of thinking of running some sql in prod to just set any thing with an entry currently there to be owned by wrfl and then we could have migration to drop this